index.html

```diff
@@ -182,7 +184,7 @@ <h1 class="gameCardTitle" id="games_genre_rank_1_title">Title #1</h1>
                                             </div>
                                         </div>
     
-                                        <div style="height: 95px; background-color: #ffd700;" class="podiumStep">1</div>
+                                        <div style="background-color: #ffd700;" class="podiumStep">1</div>

```

Comment:
The problem that there is no difference between the 1st, 2nd and third place, You have to let the size in the style here.

Reply:
fixed